packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -650,7 +650,10 @@ export class ChatController {
 
         const request = triggerPayloadToChatRequest(triggerPayload)
         const session = this.sessionStorage.getSession(tabID)
-        getLogger().info(`request from tab: ${tabID} conversationID: ${session.sessionIdentifier} request: %O`, request)
+        //eslint-disable-next-line aws-toolkits/no-json-stringify-in-log
+        getLogger().info(
+            `request from tab: ${tabID} conversationID: ${session.sessionIdentifier} request: ${JSON.stringify(request)}`
```

Comment:
how deeply nested is it? can we adjust the format() call instead, maybe increase the depth. why does this need to print an unlimited depth?